dbld/prepare-release

```diff
@@ -6,8 +6,12 @@ cd /source
 VERSION=$1
 
 OLD_VERSION=`cat VERSION`
-OLD_MAJOR=$(echo $OLD_VERSION | cut -d. -f1-2)
-NEW_MAJOR=$(echo $VERSION | cut -d. -f1-2)
+OLD_MAJOR=$(echo $OLD_VERSION | cut -d. -f1)
+OLD_MINOR=$(echo $OLD_VERSION | cut -d. -f2)
+OLD_PATCH=$(echo $OLD_VERSION | cut -d. -f3)
```

Comment:
I like the symmetry in the code (also extracting PATCH from the version number), but unfortunately PATCH is never used. Could you give an example for that?

Reply:
Yes, I extracted `PATCH` for the sake of completeness. I did not want to split the `VERSION` to 3 parts, then only provide 2 of them with variables.